xds/third_party/envoy/import.sh

```diff
@@ -135,4 +140,4 @@ do
 done
 popd
 
-popd
+popd
```

Comment:
Did you deleted the newline at the end of the file?

Reply:
Oops, my bad. I'll change it back!